torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5854,9 +5855,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            aten_name='linalg_cholesky_ex',
            dtypes=floating_and_complex_types(),
            check_batched_gradgrad=False,
+           supports_forward_ad=True,
            sample_inputs_func=sample_inputs_linalg_cholesky,
            gradcheck_wrapper=gradcheck_wrapper_hermitian_input,
-           decorators=[skipCUDAIfNoMagmaAndNoCusolver, skipCUDAIfRocm, skipCPUIfNoLapack]),
+           decorators=[skipCUDAIfNoMagmaAndNoCusolver, skipCUDAIfRocm, skipCPUIfNoLapack],
+           skips=(
+               # Gradcheck for complex generates invalid inputs for this function
```

Comment:
Might this be because it does not create Hermitian inputs? It's not clear whether we can assume that the inputs are Hermitian or not. See https://github.com/pytorch/pytorch/pull/62163#discussion_r676532573

Reply:
Ho yes, I encountered that one but forgot to fix it.
We should update https://github.com/pytorch/pytorch/blob/b73d7597089c6d27f3b3894e11d9266bd427d1e6/torch/autograd/gradcheck.py#L925 to make it keep the original imag/real respectively instead of setting them to 0s.